JASP-Engine/JASP/po/R-JASP.pot

```diff
@@ -1999,7 +2002,7 @@ msgstr ""
 msgid "P(incl|data)"
 msgstr ""
 
-msgid "Compares models that contain the effect to equivalent models stripped of the effect. Higher-order interactions are excluded. Analysis suggested by Sebastiaan Mathôt."
+msgid "Compares models that contain the effect to equivalent models stripped of the effect. Higher-order interactions are excluded. Analysis suggested by Sebastiaan Math\303\264t."
```

Comment:
You're again saving things as ASCII. I think this can also cause problems with the .po?

Reply:
HI @TimKDJ 
The strange thing is that the R-nl.po utf-8 encoding ('and has the good Mathôt') and the R-JASP.pot has us-ascii encoding
If I insert 'Mathôt' in the .pot file it automatically gets an utf-8 encoding when I save it.
I do have the feeling that the end result is ok because the result in the translated R still looks fine.
I look furter.

Reply:
@TimKDJ 
On Windows I have a similar problem: 
.pot encoding  iso-8859-1
.po encoding utf-8
You're newly generated .pot file gets automatically an utf-8 encoding, you're sure about that?

Reply:
On MacOS -- yes; because it's my system default encoding. On Windows: no, because it's not my default.

Reply:
Can you check you Locale on your Mac on my machine it is en_NL ?
Sorry that was not my intention that smily (pressed somewhere wrong)

Reply:
The only solution I can think of now is save the .pot file manually after generation with an utf-8 encoding.

Reply:
Or you switch windows to utf-8!
https://stackoverflow.com/questions/56419639/what-does-beta-use-unicode-utf-8-for-worldwide-language-support-actually-do

Reply:
(I did not test it though)

Reply:
Will merge this now, but after 0.12 ASCII .pots can't go in anymore